CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.4] - November 22, 2022
```

Comment:
We can remove dates from change log

Reply:
As you observed we followed one pattern of adding date in earlier versions, so we should add date.

Reply:
Resolved